skelebot/systems/scaffolding/scaffolder.py

```diff
@@ -20,13 +24,40 @@ def __format_variables(self, text):
                 text = text.replace(key_str, value)
 
         return text
-    
-    def __load_template(self, path):
+
+    def __format_config(self, config):
+        return eval(self.__format_variables(str(config)))
+
+    def __load_git_template(self, url):
+        # Construct template_name (folder name) and template path
+        template_name = re.sub("[:/\\.@-]+", "_", url)
+        path = TEMPLATE_PATH.format(name="")
+        path = os.path.join(os.path.dirname(__file__), path)
+
+        # Clone or pull the template from the Git URL into the template folder
+        if (os.path.exists(f"{path}/{template_name}")):
+            # If the template has been cloned before, just pull --rebase
+            status = call("git pull --rebase", shell=True)
+        else:
+            # If the template has NOT been cloned before, clone it
+            status = call(f"cd {path} && git clone {url} {template_name}", shell=True)
```

Comment:
you can simplify this section a bit using `path = TEMPLATE_PATH.format(name=template_name)` and `call(f"git clone {url} {path}", shell=True)`

Reply:
Good point!